src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -74,6 +74,9 @@ protected function setDefaults(
         $searchNavigationRequest->setUserIp($this->fetchClientIp());
         $searchNavigationRequest->setRevision($this->getPluginVersion());
         $searchNavigationRequest->setOutputAdapter(OutputAdapter::XML_21);
+        $searchNavigationRequest->addIndividualParam('shopType', 'Shopware6', true);
+        $searchNavigationRequest->addIndividualParam('shopVersion', Utils::getCleanShopwareVersion(), 'set');
```

Comment:
In the docblock of `addIndividualParam` it clearly states:

>   * @param $method string Use Request::ADD_VALUE to add the param and not overwrite existing ones and
      Request::SET_VALUE to overwrite existing params.

Why using a hardcoded string when have the constant `Request::SET_VALUE`?